### DISCUSSION TITLE
“Splitting points” for a scene?

### DISCUSSION THREAD

**Discussion by jaybonthius** [ManimCommunity/manim#2510] [Architecture]
I'd like to “split” my scene into multiple video files (for importing into a slides presentation). I understand that each `self.play(...)` makes a `partial_movie_file`, but that doesn't solve my problem: it would generate too many files for loop situations, and the filenames are randomly generated. Here's a minimal working example:

```py
class example( Scene ):
    def construct(self):
        c = Circle()
        # video one
        self.play( Create(c) )
        # video two
        for i in range(10):
            self.play(c.animate.set_color(random_color()))
```

Currently, this renders 11 partial movie files and one complete file `example.mp4`. I'd like to generate **two** videos from this scene: one where the circle is created, and one for the entire loop of changing colors, both named in sensible ways (e.g. `example_1.mp4` and `example_2.mp4`. Is there some way to do this programmatically? Some sort of `self.split()` method?

**Comment by behackl** [comment]
Take a look at Sections here: https://docs.manim.community/en/stable/tutorials/a_deeper_look.html#sections

**Reply by jaybonthius** [reply]
@behackl holy moly that's exactly what I'm looking for! Thanks a bunch.
